SeaPublicWebsite.Data/SeaPublicWebsite.Data.csproj

```diff
@@ -6,7 +6,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.5" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.4" />
```

Comment:
Any particular reason for this downgrade?

Reply:
The PostGres library uses 6.0.4 so we get a clash and a build warning (and it uses 6.0.4 anyway to resolve things)